FindYourRecipe/FindYourRecipe.Contracts/IIngredientRecipeService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using FindYourRecipe.Contracts.Models;
+
+namespace FindYourRecipe.Contracts
+{
+	public interface IIngredientRecipeService
+	{
+		Task DeleteAsync(int id);
+
+		Task<IngredientRecipeResponseModel> CreateAsync(CreateIngredientRecipeRequestModel request);
+
+		Task<IngredientRecipeResponseModel> UpdateAsync(int id,CreateIngredientRecipeRequestModel request);
```

Comment:
Should the second parameter type be `CreateOrUpdateIngredientRecipeRequestModel`?